Mathlib/Topology/MetricSpace/Basic.lean

```diff
@@ -60,23 +60,13 @@ universe u v w
 
 variable {α : Type u} {β : Type v} {X ι : Type _}
 
-/-- Construct a uniform structure core from a distance function and metric space axioms.
-This is a technical construction that can be immediately used to construct a uniform structure
-from a distance function and metric space axioms but is also useful when discussing
-metrizable topologies, see `pseudo_metric_space.of_metrizable`. -/
-def UniformSpace.Core.ofDist {α : Type _} (dist : α → α → ℝ) (dist_self : ∀ x : α, dist x x = 0)
-    (dist_comm : ∀ x y : α, dist x y = dist y x)
-    (dist_triangle : ∀ x y z : α, dist x z ≤ dist x y + dist y z) : UniformSpace.Core α :=
-  .ofFun dist dist_self dist_comm dist_triangle $ fun ε ε0 => ⟨ε / 2, half_pos ε0, fun _ h₁ _ h₂ =>
-    add_halves ε ▸ add_lt_add h₁ h₂⟩
-#align uniform_space.core_of_dist UniformSpace.Core.ofDist
-
```

Comment:
If you're forward-porting this, can you update the SHAs too? And preferably insert a commit with the updated mathport output so that this is diffable.

Reply:
I'll do this before making the pr as awaiting review.